### PR TITLE
added initial files

### PR DESCRIPTION
**Pull request by nimdolphin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// Задания условное ветвление: if, ?
+
+// Используя конструкцию if..else, напишите код, который будет спрашивать: „Какое «официальное» название JavaScript?“
+let javaScript = prompt("Какое «официальное» название JavaScript?", "");
+
+if (javaScript == "ECMAScript") {
```
Comment (kathrynowy) [inline index.js]: strcit comparison

File: index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+} else {
+  console.log(0);
+}
+
+// Перепишите конструкцию if с использованием условного оператора '?':
+let result = a + b < 4 ? "Мало" : "Много";
+
+// Перепишите if..else с использованием нескольких операторов '?'.
+let message =
+  login == "Сотрудник"
```
Comment (kathrynowy) [inline index.js]: strict comparison

File: index.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    : login == "Директор"
+    ? "Здравствуйте"
+    : login == ""
+    ? "Нет логина"
+    : "";
+
+// 1. Если переменная b равна 10, то выведите 'Верно', иначе выведите 'Неверно'.
+let b = 10;
+
+if (b == 10) {
```
Comment (kathrynowy) [inline index.js]: same, check everywhere

File: index.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+} else if (min >= 30 && min <= 44) {
+  console.log("the third part");
+} else if (min >= 45 && min <= 59) {
+  console.log("the fourth part");
+}
+
+// 3. Если переменная c равна нулю, то выведите 'Верно', иначе выведите 'Неверно'. Проверьте работу скрипта при c, равном 1, 0, -3.
+let c = prompt("введите число", "");
+
+if (c == 0) {
```
Comment (kathrynowy) [inline index.js]: we don't check like this

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 2 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 3 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 4 above)
Comment (nimdolphin) [reply index.js]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**